CryptoTechReminderSystem.Test/Gateway/HarvestGatewayTests.cs

```diff
@@ -127,7 +141,7 @@ public void CanSendOneRequestAtATime()
 
                 _harvestGateway.RetrieveTimeSheets(_defaultDateFrom, _defaultDateTo);
                
-                _harvestApi.ReceivedRequests.Count.Should().Be(1);
```

Comment:
Should we have a test with one page of data as well as the test with two pages?

Reply:
Agreed with Robert. There should be multiple tests for maximum coverage.

Reply:
Resolved in https://github.com/chubberlisk/crypto_tech/pull/33/commits/7a14902931aef968d2d0de1e54922358413c56b5.